### DISCUSSION TITLE
How can I check my rank?

### DISCUSSION THREAD

**Discussion by Dominik48N** [anuraghazra/github-readme-stats#2070] [Q&A] [answered]
**Hey,**
I wanted to ask how I can check where I stand in the ranking. I know S, S+, A++ etc., but I would like to know, for example, what percentage I still need for the S rank.

_I hope my question is a little understandable._

**Comment by rickstaa** [comment, marked as answer]
@Dominik48N You can find the rank calculation at https://github.com/anuraghazra/github-readme-stats/blob/master/src/calculateRank.js. There are, however, some proposals to improve the ranking since it is really hard to get a different rank than A+ (see https://github.com/anuraghazra/github-readme-stats/issues/455). We will probably merge https://github.com/anuraghazra/github-readme-stats/pull/1186 to fix this in the coming months.

**Reply by Dominik48N** [reply]
Okay, Thank you for the reply.
